src/models.py

```diff
@@ -36,3 +43,15 @@ class UserSession(Base):  # pylint: disable=too-few-public-methods
     id = Column(Integer, primary_key=True)  # noqa: A003
     session_id = Column(String, nullable=False, unique=True)
     user_id = Column(Integer, ForeignKey("users.id"), nullable=False)
+
+
+class Topic(Base):  # pylint: disable=too-few-public-methods
+    """A model class for topics table."""
+
+    __tablename__ = "topics"
+
+    id = Column(Integer, primary_key=True)  # noqa: A003
+    author_id = Column(Integer, ForeignKey("users.id"), nullable=False)
+    created_at = Column(DateTime(timezone=False), server_default=func.now())
```

Comment:
I think `timezone` is `False` by default, so we can just remove it:
```python
created_at = Column(DateTime, server_default=func.now())
```

Reply:
And we need `nullable=False` here as well. Since every post should have creation time.

Reply:
There is a default value in that column. Does it not make `nullable=False` redundant since default value is provided every time?

Reply:
Yes, this column has a default value, and if someone will create a new row in that table, and will not fill `created_at` column, it will be filled automatically. But we need to define an explicit constraint to enforce that this column can never be `NULL`. Why? Because someone - probably unintentionally - can put `NULL` in that column by hand or by script or it may be a bug in our code, which will accidentally write `NULL` values. And if we didn't define `nullable=False` then we cannot be _actually sure_ that there is no `NULL` values in this column.